rasa/core/featurizers/tracker_featurizers.py

```diff
@@ -83,7 +86,18 @@ def _featurize_states(
         self,
         trackers_as_states: List[List[State]],
         interpreter: NaturalLanguageInterpreter,
-    ) -> List[List[Dict[Text, List["Features"]]]]:
+    ) -> List[List[Dict[Text, List[Features]]]]:
+        """Featurizes state histories with `state_featurizer`.
+
+        Args:
+            trackers_as_states: Lists of states produced by a `DialogueStateTracker`
+                instance.
+            interpreter: A `NaturalLanguageInterpreter` for the `state_featurizer`
```

Comment:
@kedz I see data types mentioned in the arg descriptions and return descriptions of some functions of this file. Can you please double check across the whole PR and remove those?

Reply:
On it!

Reply:
Ok any mentions to class types in docstrings should be in reference to things not already defined in the type annotations but let me know if you see anything else that should be changed.